assets/pages/contributions-landing/actions/contributionsLandingActions.js

```diff
@@ -12,6 +12,7 @@ export type Action =
   | { type: 'CHANGE_CONTRIB_AMOUNT', amount: Amount }
   | { type: 'CHANGE_CONTRIB_AMOUNT_RECURRING', amount: Amount }
   | { type: 'CHANGE_CONTRIB_AMOUNT_ONEOFF', amount: Amount }
+  | { type: 'SET_CONTEXT', context: boolean }
```

Comment:
Instead of a boolean, would it make sense for this is a string/enum that indicates the amount of context the user already has?

Reply:
Context here refers to whether we include the contextual copy, whether we give the user 'context' for there donation. There is probably a better word...